packages/metro-bundler/src/Resolver/__tests__/Resolver-test.js

```diff
@@ -219,14 +218,14 @@ describe('Resolver', function() {
       const module = createModule('test module', ['x', 'y']);
       const resolutionResponse = new ResolutionResponseMock({
         dependencies: [module],
-        mainModuleId: 'test module',
+        mainModuleId: 'test module'
```

Comment:
In the internal lint/prettier config, we force adding the commas, so it wouldn't be compatible. We should change prettier configuration instead, it seems.